Project/src/main/resources/static/js/upload.js

```diff
@@ -0,0 +1,20 @@
+function checkFileSize(event) {
+    var tenMegabytes = 10000000; //10 MB in bytes
+    var fileElement = document.getElementById("fileLoad");
+    var fileArr = fileElement.files;
+
+    if (!fileArr || !fileArr[0]) {
+        alert("No file loaded");
```

Comment:
not sure an alert is the best

Reply:
What do you suggest then? This literally just makes a popup that there's no file and then they click ok and then they can add in the file.

Reply:
I mean you can do it like that but i think having the alert on the page is better. @Taz-Z can you weigh in.

Reply:
I think that's fine if you want to do it on the page, but can we wait until we integrate bootstrap? I think errors are more graceful on there.

Reply:
I say keep it as is for now - wait till bootstrap integration and think of a "nicer" sln